test/tools/ossfuzz/config/solidity.dict

```diff
@@ -0,0 +1,210 @@
+"continue"
+"delete "
+"do"
+"else"
+"for (a=0;a<2;a++)"
+"function bid() public payable {"
+"modifier onlySeller() {"
+"if"
+"new "
+"return"
+"switch"
+"throw"
+"try"
+"while"
+"pragma solidity ^90.90.0"
+"pragma experimental ABIEncoderV2;"
+"pragma experimental SMTChecker;"
+"pragma solidity >=0.4.0 <0.6.0;"
+"x << y"
+"x * 2**y"
+"x / 2**y"
+"int x = -2**255;"
+"x % y"
+"0**0"
+"<<"
+">>"
+"|"
+"&"
+"^"
+"~"
+"<="
+"<"
+"=="
+"!="
+">="
+">"
+"+"
+"-"
+"*"
+"/"
+"%"
+"**"
+"fixed "
+"unfixed "
+"ufixed128x128 "
+"fixed128x128 "
+"float "
+"double "
+".balance"
+".transfer("
+".send("
+"true "
+"false "
+".call("
+".callcode("
+".delegatecall( "
+".gas("
+".value("
+"this"
+"2e10"
+"1.1"
+"hex"001122FF""
+"bool "
+"byte "
+"bytes1 "
+"bytes2 "
+"bytes3 "
+"bytes4 "
+"bytes5 "
+"bytes6 "
+"bytes7 "
```

Comment:
Do we need to list all of them? Perhaps just the interesting edge cases?

Reply:
What would the corner cases be, for example?